src/main/java/frc/robot/vision/sources/VisionSource.java

```diff
@@ -0,0 +1,18 @@
+package frc.robot.vision.sources;
+
+import frc.robot.vision.filters.Filter;
+import frc.robot.vision.rawdata.RawVisionData;
+
+import java.util.Optional;
+
+public interface VisionSource<T extends RawVisionData> {
+
+	void updateEstimation();
+
+	Optional<T> getRawVisionEstimation();
+
+	boolean shallBeFiltered();
```

Comment:
what is this doing?

Reply:
Returning if the latest given observation should be filtered

Reply:
Use better english